### ISSUE TITLE
Get check_video_stream_images working with current tools

### ISSUE THREAD

**Issue by petterreinholdtsen** [Frikanalen/frikanalen#137]
The nagios check check_video_stream_images do not work with current mplayer, as the command line arguments seem to have changed.  We should get it working again.

In short, the check connect to a video stream, take a snap shot, wait 30 seconds and take another snapshot.  If the two images are very similar, something is wrong and the check report an error.

**Comment by petterreinholdtsen** [comment]
Perhaps we can use vlc?  I came across this recipe for taking a snapshot using vlc:

```
cvlc http://video.nuug.no/frikanalen.webm --rate=1 --video-filter=scene --vout=dummy \
  --start-time=0 --stop-time=0.1 --scene-format=png --scene-ratio=24 \
  --scene-prefix=snap --scene-path=`pwd` vlc://quit
```

**Comment by petterreinholdtsen** [comment]
This was fixed in commit d4d00419dd91f1669ebc36f3a9526573e41a4593.
